apps/examples/testcase/le_tc/kernel/tc_sfs.c

```diff
@@ -0,0 +1,930 @@
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @fn                   :tc_fwrite_data16
+ * @brief                :using fwrite on 16 bytes data.
+ * API's covered         :fwrite, fread
+ * Input         	  :"abcdefgh12345678"
+ * @return               :void
+ */
+
+void tc_fwrite_data16(void)
+{
+	char *input = "abcdefgh12345678";
+	size_t len;
+
+	memset(data, '\0', sizeof(data));
+	len = strlen(input);
+	file_writetest_fwrite(input, len);
+	file_readtest_fread(data);
+	TC_ASSERT_EQ("tc_fwrite_data16", strcmp(input, data), 0);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @fn                   :tc_fwrite_data24
+ * @brief                :using fwrite on 24 bytes data.
+ * API's covered         :fwrite, fread
+ * Input         	  :"abcdefgh12345678abcdefgh12345678"
+ * @return               :void
+ */
+
+void tc_fwrite_data24(void)
+{
+	char *input = "abcdefgh12345678abcdefgh";
+	size_t len;
+
+	memset(data, '\0', sizeof(data));
+	len = strlen(input);
+	file_writetest_fwrite(input, len);
+	file_readtest_fread(data);
+	TC_ASSERT_EQ("tc_fwrite_data24", strcmp(input, data), 0);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @fn                   :tc_fwrite_data121
+ * @brief                :using fwrite on 121 bytes data.
+ * API's covered         :fwrite, fread
+ * Input         	  :"abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh1"
+ * @return               :void
+ */
+
+void tc_fwrite_data121(void)
+{
+	char *input = "abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh1";
+	size_t len;
+
+	memset(data, '\0', sizeof(data));
+	len = strlen(input);
+	file_writetest_fwrite(input, len);
+	file_readtest_fread(data);
+	TC_ASSERT_EQ("tc_fwrite_data121", strcmp(input, data), 0);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @fn                   :tc_fwrite_data255
+ * @brief                :using fwrite on 255 bytes data.
+ * API's covered         :fwrite, fread
+ * Input         	  :"abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh1234567"
+ * @return               :void
+ */
+
+void tc_fwrite_data255(void)
+{
+	char *input = "abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh12345678abcdefgh1234567";
+	size_t len;
+
+	memset(data, '\0', sizeof(data));
+	len = strlen(input);
+	file_writetest_fwrite(input, len);
+	file_readtest_fread(data);
+	TC_ASSERT_EQ("tc_fwrite_data255", strcmp(input, data), 0);
+	TC_SUCCESS_RESULT();
+
+}
+
+/**
+ * @fn                   :tc_fprintf_data2
+ * @brief                :using fprintf on 2 bytes data.
+ * API's covered         :fprintf, fread
```

Comment:
You can mention all API's covered like:
fopen, remove, fclose
Also you can do same for other TC's